src/libraries/System.Runtime.InteropServices.JavaScript/gen/JSImportGenerator/JSTypeFlags.cs

```diff
@@ -22,6 +22,7 @@ internal enum JSTypeFlags : int
         MemoryView = 0x800,
         Any = 0x1000,
         Discard = 0x2000,
+        OneWay = 0x4000,
```

Comment:
This probably should have a different name

Reply:
Do you have any suggestions ?

Reply:
I would prefer we not encode this using JSType, but if we do, this should be a superset of Discard/Void, i.e. `DiscardNoWait` or something like that.